actions-server/src/handlers/joinGame.ts

```diff
@@ -0,0 +1,86 @@
+import { Request, Response } from "express"
+import { DocumentNode } from "graphql"
+import jwt from "jsonwebtoken"
+import fetch from "node-fetch"
+import {
+  InsertPlayerForGameDocument,
+  InsertPlayerForGameMutationVariables,
+  LookupPlayerForGameDocument,
+  LookupPlayerForGameQueryVariables,
+} from "../../../app/src/generated/graphql"
```

Comment:
a bit strange, reaching out of this project and into another... maybe we could move the generated files into a whole new subdir. And have both typescript apps (react, express) reference that?